awscli/examples/controltower/untag-resource.rst

```diff
@@ -0,0 +1,10 @@
+**To Untag Control Tower Enabled Control**
+
+The following ``untag-resource`` example untags an AWS Control Tower enabled control::
+
+    aws controltower untag-resource \
+        --resource-arn "arn:aws:controltower:us-east-1:123456789012:enabledcontrol/2H2AWUG4SKG81855" \
+        --tag-keys "TestTagKey"
+
```

Comment:
Missing output section. Please provide output or add the line ``This command produces no output.``
https://aws.github.io/aws-cli/docs_styleguide.html#output